sources/academy/platform/getting_started/creating_actors.md

```diff
@@ -69,23 +69,23 @@ await Actor.exit();
 To make this simple Actor functional, let's delete the current value of `result`, and replace it with the numbers we are getting from the user's input:
 
 ```js
-const result = input?.firstNumber + input?.secondNumber;
+const result = (input?.firstNumber || 0) + (input?.secondNumber || 0);
 ```
 
 The final code should look like this:
 
 ```js
 import { Actor } from 'apify';
 
-await Actor.init()
+await Actor.init();
 
 console.log('Loading input');
 
 const input = await Actor.getInput();
 console.log('First number: ', input?.firstNumber);
 console.log('Second number: ', input?.secondNumber);
 
-const result = input?.firstNumber + input?.secondNumber;
+const result = (input?.firstNumber || 0) + (input?.secondNumber || 0);
```

Comment:
its a bit ugly when condensed into a single line, what about this? note that we want to use `??` here if something, not `||`

```suggestion
const firstNumber = input?.firstNumber ?? 0;
const secondNumber = input?.secondNumber ?? 0;
const result = firstNumber + secondNumber;
```